roles/ssh_jumper/defaults/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_ssh_jumper"
+
+cifmw_ssh_jumper_config:
+  target: "{{ inventory_hostname }}"
```

Comment:
so now it's a list of dict

Reply:
right, and that makes the defaults handling a bit more involved.

I guess we should have a vars/main.yaml with `cifmw_ssh_jumper_config_defaults` - and then merge each entry in list with that.